packages/terra-show-hide/README.md

```diff
@@ -0,0 +1,26 @@
+# Terra Show Hide
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-show-hide.svg)](https://www.npmjs.org/package/terra-show-hide)
```

Comment:
Update to be:
```
[![NPM version](https://badgen.net/npm/v/terra-show-hide)](https://www.npmjs.org/package/terra-abstract-modal)
[![Build Status](https://badgen.net/travis/cerner/terra-core)](https://travis-ci.org/cerner/terra-core)
```

Reply:
https://github.com/cerner/terra-core/pull/1910/commits/3123a966699da53c55bd73f6c69a7aad5d33b8f4